pep-0011.txt

```diff
@@ -99,16 +99,41 @@ Tier 2
   reverted within 24 hours**.
 - Failures on these platforms **block a release**.
 
-============================= ========================== ========
-Target Triple                 Notes                      Contacts
-============================= ========================== ========
-aarch64-apple-darwin          clang                      Ned Deily, Ronald Oussoren, Dong-hee Na
-aarch64-unknown-linux-gnu     glibc, gcc                 Petr Viktorin, Victor Stinner
-
-                              glibc, clang               Victor Stinner, Gregory P. Smith
-powerpc64le-unknown-linux-gnu glibc, gcc                 Petr Viktorin, Victor Stinner
-x86_64-unknown-linux-gnu      glibc, clang               Victor Stinner, Gregory P. Smith
-============================= ========================== ========
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Target Triple
+     - Details
+     - Buildbot
```

Comment:
I purposefully pulled out the buildbot links as those builder numbers are not stable, so I would prefer to not add them back in.